frontend/src/app/testQueue/addToQueue/AddToQueueSearch.tsx

```diff
@@ -48,6 +48,10 @@ export const QUERY_PATIENT = gql`
       birthDate
       gender
       telephone
+      phoneNumbers {
```

Comment:
**question:** Does this introduce an N+1 query?

Reply:
See af59342 for an attempt to address this by adding a data loader for patient phone numbers.

Reply:
Looks good!